Missionframework/scripts/server/battlegroup/spawn_battlegroup.sqf

```diff
@@ -55,8 +55,8 @@ if !(_spawn_marker isEqualTo "") then {
             sleep 0.5;
 
             (crew _vehicle) joinSilent _nextgrp;
-            [_nextgrp] spawn battlegroup_ai;
-            _bg_groups pushback _nextgrp;
+            [_nextgrp] call battlegroup_ai;
+            _nextgrp setVariable ["KPLIB_isBattleGroup", true];
 
```

Comment:
You've removed the following line, this means the battlegroup won't be transferred to HC at end of this script. I don't think this is intentional.
```suggestion
_bg_groups pushback _nextgrp;

```

Reply:
That is indeed not intentional. Good catch!